src/Caliburn.Micro.Extensions.WIN8/AppManifestHelper.cs

```diff
@@ -75,6 +75,11 @@ public Windows.UI.Color BackgroundColor
 
         private static Windows.UI.Color ToColor(string hexValue)
         {
+            // if 'transparent' is entered in the app manifest, return Windows.UI.Colors.Transparent
+            // in order to prevent parsing failures
+            if (hexValue == "transparent")
```

Comment:
You might want to make comparison case insensitive to handle `Transparent` value.